plone-5.2.x.cfg

```diff
@@ -4,3 +4,7 @@ extends =
     https://dist.plone.org/release/5.2.1/versions.cfg
 find-links += https://dist.plone.org/thirdparty/
 versions=versions
+
+[versions]
+# fixes Error: The requirement ('importlib-metadata>=1') is not allowed by your [versions] constraint (0.20)
+importlib-metadata = 1.7.0
```

Comment:
I think this is not supported in Py2.7.
You might want to add a:
```
[versions:python27]
importlib-metadata = 1.6.1
```
I found that in one of ours py2.7 projects.